src/containers/status/Status.js

```diff
@@ -18,7 +18,7 @@ import { useSelector } from 'react-redux';
 import truncate from 'truncate-middle';
 import { Dvr, GitHub } from '@material-ui/icons';
 
-import { selectConnection, selectByzantine, selectIsSynced } from 'selectors/statusSelector';
+import { selectConnection, selectByzantine, selectIsSynced} from 'selectors/statusSelector';
```

Comment:
sorry nit, but add back that space 😆

Reply:
can you add `"object-curly-spacing": ["error", "always"],` to `.eslintrc.json` rules